protected/humhub/modules/content/models/Content.php

```diff
@@ -409,6 +409,16 @@ public function isArchived()
      */
     public function canArchive()
     {
+        // Currently global content can not be archived
+        if(!$this->contentcontainer_id) {
+            return false;
+        }
+
+        // No need to archive content on an archived container, content is marked as archived already
+        if($this->content->content->getContainer()->isArchived()) {
```

Comment:
@buddh4 Can you please check your codestyle settings in the IDE? Actually, there should be a whitespace after the `if`.

Reply:
Not an issue with the ide, just did not auto format. Not really used to the space after such blocks.